lessons/Early-Math/associative-property/langs/en-us/pages/assoc-prop-table.adoc

```diff
@@ -0,0 +1,49 @@
+= Associative Property Table
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+</style>
+++++
+
+
+For each Circle of Evaluation, apply the Associative Property to create two equivalent Circles. Make sure you use different _groupings_ in each Circle. Note: There are multiple possible responses here! @ifsoln{Some possible responses are shown below. Your students will likely come up with others.}
```

Comment:
This is one of the pages where I wondered if the left column should be an expression rather than a coe

Reply:
I just tried reformatting this. It didn't feel right. Looked out-of-balance. Leaving it how it was. 
Shrug.